ui/app/accounts/import/index.js

```diff
@@ -37,6 +38,14 @@ AccountImportSubview.prototype.render = function () {
       style: {
       },
     }, [
+      h('.section-title.flex-row.flex-center', [
+        h('i.fa.fa-arrow-left.fa-lg.cursor-pointer', {
+          onClick: (event) => {
+            props.dispatch(actions.goHome())
```

Comment:
In the video it looks like it abruptly jumped back a screen. You should be able to make it transition backwards by adding the line `props.dispatch(actions.transitionBackward())` before the `goHome()` call here.

Does it actually look jumpy on yours, or was that just the gif?

Nice gif + PR btw, the gif PR is like the coolest way to submit a feature.